Install the PAM service in the configured system directory, not /usr/etc.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -162,7 +162,7 @@
 install(TARGETS abysslock)
 
 install(FILES ${CMAKE_SOURCE_DIR}/pam/abysslock
-        DESTINATION ${CMAKE_INSTALL_PREFIX}/etc/pam.d)
+        DESTINATION ${CMAKE_INSTALL_SYSCONFDIR}/pam.d)
 
 install(
   FILES ${CMAKE_SOURCE_DIR}/assets/example.conf
